treeherder/webapp/api/note.py

```diff
@@ -67,13 +78,33 @@ def create(self, request, project):
         """
         POST method implementation
         """
+        j = Job.objects.get(repository__name=project, id=int(request.data['job_id']))
+        fc_id = int(request.data['failure_classification_id'])
         JobNote.objects.create(
-            job=Job.objects.get(repository__name=project, id=int(request.data['job_id'])),
-            failure_classification_id=int(request.data['failure_classification_id']),
+            job=j,
+            failure_classification_id=fc_id,
             user=request.user,
             text=request.data.get('text', ''),
         )
 
+        if fc_id == 2:
+            # remove cached failure line counts
+            line_cache_key = 'error_lines'
+            line_cache = cache.get(line_cache_key)
+            date = '%s' % j.submit_time
```

Comment:
This and the following line can be written as `date = j.submit_time.date().iso_format()`.